signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +410,28 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_signing_service_daily_transaction_sum_so_far() -> int:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds')  # pylint: disable=no-member
+        thresholds_directory.mkdir(exist_ok=True)  # pylint: disable=no-member
+        daily_file = datetime.datetime.now().strftime('%Y-%m-%d')
+        thresholds_directory.joinpath(daily_file).touch(exist_ok=True)  # pylint: disable=no-member
+        thresholds_directory = thresholds_directory.joinpath(daily_file)
+        if thresholds_directory.read_text() == '':  # pylint: disable=no-member
+            thresholds_directory.write_text('0')  # pylint: disable=no-member
+            return 0
+        try:
```

Comment:
Quite frankly, I would simplify this:
Just try to read from: `daily_file` and convert it to `int`. Catch any exception from the combined operation and set `self.daily_transactions_sum = 0`

Reply:
Done